🐛 [Bug]: serverStoryblokClient: Cannot find module '#storyblok/server' or its corresponding type declarations.

### Package

@storyblok/nuxt (Nuxt Module)

### Bug Description

When I try to import { serverStoryblokClient } from '#storyblok/server' as suggested in https://www.storyblok.com/mp/better-nitro-support-improved-security-nuxt-sdk I get:
`Cannot find module '#storyblok/server' or its corresponding type declarations. `

### Steps to Reproduce

1. in the Nuxt config enableServerClient: true
2. create a API route.
3. import { serverStoryblokClient } from '#storyblok/server';

### Expected Behavior

module not missing

### Actual Behavior

even tho it works, I get runtime errors because of the type declaration is missing

### Code Sample

```javascript
import type { H3Event } from 'h3';
import { serverStoryblokClient } from '#storyblok/server';

export default defineEventHandler(async (event: H3Event) => {
  const slug = getRouterParam(event, 'slug');
  const storyblokApi = serverStoryblokClient(event);
  const { data } = await storyblokApi.get(`cdn/stories/${slug}`, {
    version: 'draft',
    resolve_links: 'url',
  });
  return { story: data.story };
});
```

### Environment

```shell
System:
    OS: macOS 26.2
    CPU: (10) arm64 Apple M1 Max
    Memory: 148.42 MB / 32.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 22.18.0 
    Yarn: 1.22.22 
    npm: 11.9.0 
    pnpm: 8.9.2
  Browsers:
    Chrome: 144.0.7559.133
    Edge: 144.0.3719.115
    Firefox: 134.0.2
    Safari: 26.2
    Safari Technology Preview: 26.0
  npmPackages:
    @storyblok/nuxt: 9.0.0 => 9.0.0 
    nuxt: 4.3.0 => 4.3.0 
    vue: 3.5.27 => 3.5.27
```

### Error Logs

```shell
[vue-tsc] Cannot find module '#storyblok/server' or its corresponding type declarations.
…/server/api/storyblok/[slug].get.ts:2:39
    1 | import type { H3Event } from 'h3';
  > 2 | import { serverStoryblokClient } from '#storyblok/server';
      |                                       ^^^^^^^^^^^^^^^^^^^
    3 |
    4 | export default defineEventHandler(async (event: H3Event) => {
    5 |   const slug = getRouterParam(event, 'slug');
```

### Additional Context

_No response_

--- a/packages/nuxt/package.json
+++ b/packages/nuxt/package.json
@@ -17,6 +17,10 @@
     ".": {
       "types": "./dist/types.d.mts",
       "import": "./dist/module.mjs"
+    },
+    "./server": {
+      "types": "./dist/runtime/server/index.d.ts",
+      "import": "./dist/runtime/server/index.js"
     }
   },
   "main": "./dist/module.mjs",

--- a/packages/nuxt/src/module.ts
+++ b/packages/nuxt/src/module.ts
@@ -67,6 +67,7 @@ export default defineNuxtModule<AllModuleOptions>({
       nitroConfig.alias = nitroConfig.alias || {};
       nitroConfig.alias['#storyblok/server'] = resolver.resolve('./runtime/server');
     });
+    nuxt.options.alias['#storyblok/server'] = resolver.resolve('./runtime/server');
 
     // Add auto imports
     const names = [

